File: app/src/main/java/com/andysklyarov/finnotify/framework/database/LastDate.java
package com.andysklyarov.finnotify.framework.database;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class LastDate {

    @PrimaryKey()
    @ColumnInfo(name = "id")
    private int id;

    @ColumnInfo(name = "date")
    private String date;

    public LastDate() {
    }

    public LastDate(int id, String date) {
        this.id = id;
        this.date = date;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }
}

File: app/src/main/java/com/andysklyarov/finnotify/framework/database/LastDate.kt
package com.andysklyarov.finnotify.framework.database

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity
data class LastDate(
    @PrimaryKey()
    @ColumnInfo(name = "id") val id: Int,

    @ColumnInfo(name = "date") val date: String
)